src/DynamoCore/UpdateManager/UpdateManager.cs

```diff
@@ -434,6 +447,18 @@ public IAppVersionInfo DownloadedUpdateInfo
             }
         }
 
+        public bool IsUpdateAvailable
+        {
+            get
+            {
+                //Update is not available unitl it's downloaded
+                if(DownloadedUpdateInfo==null)
+                    return false;
+
+                return ForceUpdate || AvailableVersion > ProductVersion;
```

Comment:
Yes, except for ForceUpdate is set by the configuration or by directly updating the property from Debug UI. Just wanted to ensure that green cloud is visible only after we have DownloadedUpdateInfo